scientific_library/tvb/simulator/simulator.py

```diff
@@ -50,16 +50,24 @@
 from .history import SparseHistory
 from tvb.basic.neotraits.api import HasTraits, Attr, NArray, List, Float
 
+from tvb.simulator.models.reduced_wong_wang_exc_io_inh_i import ReducedWongWangExcIOInhI
+from tvb_multiscale.config import CONFIGURED
+
 
 # TODO with refactor, this becomes more of a builder, since iterator will account for
 # most of the runtime associated with a simulation.
 class Simulator(HasTraits):
+
+    tvb_spikeNet_interface = None
```

Comment:
This `tvb_spikeNet_interface ` seems to be a complex object, missing from this PR

Reply:
yep there is a rather large refactor to complete first.

Reply:
If this is None, it meens no multiscale simulation is going to take place, and it is just ignored.